extension-manifest-v3/src/utils/trackerdb.js

```diff
@@ -64,30 +66,35 @@ export function getMetadata(request) {
 
   if (matches.length === 0) {
     exception = getException(request.domain);
-    if (!exception) return null;
 
-    matches = [
-      {
-        pattern: { key: request.domain, name: request.domain },
-        category: { key: 'unidentified' },
-      },
-    ];
+    if (!exception && !request.blocked && !request.modified) {
+      return null;
+    }
+
+    tracker = {
+      id: request.domain,
+      name: request.domain,
```

Comment:
should those be domain or hostname in case of IP addresses?

Reply:
I'll add fallback in the `domain` property of the `ExtendedRequest`